client/state/stats/lists/test/selectors.js

```diff
@@ -90,9 +91,12 @@ describe( 'selectors', () => {
 							2916284: {
 								statsStreak: {
 									'[["endDate","2016-06-01"],["startDate","2015-06-01"]]': {
-										1461961382: 1,
-										1464110402: 1,
-										1464110448: 1
+										streak: {},
```

Comment:
I considered it - but figured since the API is returning it, it could possibly be used in a future iteration of stats ( or was likely part of an earlier design ).  But current functionality + simplicity should take precedence.  I will switch it up.